### PR TITLE
Add var and var file to destroy operation

### PR DESCRIPTION
**Pull request by luis-silva**
Added -var-file and -var to destroy operation and ran go fmt on the code.

I built this docker image to test the plugin: https://hub.docker.com/r/lsilva/drone-terraform/tags/

### CODE REVIEW

File: plugin.go [hunk 1]
```diff
@@ -255,6 +255,13 @@ func tfDestroy(config Config) *exec.Cmd {
 	for _, v := range config.Targets {
 		args = append(args, fmt.Sprintf("-target=%s", v))
 	}
+	for _, v := range config.VarFiles {
+		args = append(args, "-var-file", fmt.Sprintf("%s", v))
```
Comment (appleboy) [inline plugin.go]:
```go
fmt.Sprintf("-var-file=%s", v)
```

File: plugin.go [hunk 2]
```diff
@@ -255,6 +255,13 @@ func tfDestroy(config Config) *exec.Cmd {
...
+	}
+	for k, v := range config.Vars {
+		args = append(args, "-var")
+		args = append(args, fmt.Sprintf("%s=%s", k, v))
```
Comment (appleboy) [inline plugin.go]:
```go
args = append(args, "-var", fmt.Sprintf("%s=%s", k, v))
```

**Comment by luis-silva** [comment]
@appleboy changes applied

File: Dockerfile [hunk 3]
```diff
@@ -14,7 +14,11 @@ RUN apk -U add \
     ca-certificates \
     git \
     wget \
-    openssh-client && \
+    openssh-client \
+    python \
+    py-pip  && \
+    pip install --upgrade awscli && \
```
Comment (appleboy) [inline Dockerfile]: Why need AWS CLI tool?

Comment (luis-silva) [reply Dockerfile]: @appleboy My bad. I committed on my master and forgot this PR was still open. I'll raise this change in a separate issue for discussion.

**Pull request merged**